create_infotexts.py

```diff
@@ -8,88 +8,89 @@
 Outputs <Fotonummer>.info files in a subdirectory ./infofiles/
 """
 
-import os
 import json
 import re
 import pandas as pd
 import batchupload.helpers as helpers
 
 people_mapping_string = """{
```

Comment:
Since you are touching this string anyway.
Could you also add a TODO with the Task # for breaking this out as an external .json file?